prep-css-flexbox/justify-content.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    div {
+      display: flex;
+      justify-content: center;
+    }
+
+    .card-container {
+      flex-basis: calc(100% / 4);
+      padding: 10px;
+    }
+
+    .card-deck {
```

Comment:
This rule-set should have a selector for `#card-deck` (the element id) instead of `.card-deck` (the element class).